src/test/java/org/sagebionetworks/bridge/models/schedules2/adherence/eventstream/EventStreamTest.java

```diff
@@ -0,0 +1,110 @@
+package org.sagebionetworks.bridge.models.schedules2.adherence.eventstream;
+
+import static org.sagebionetworks.bridge.TestConstants.CREATED_ON;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+
+import org.joda.time.LocalDate;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.sagebionetworks.bridge.models.schedules2.timelines.TimelineMetadata;
+import org.testng.annotations.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class EventStreamTest {
+    
+    @Test
+    public void canSerialize() throws Exception {
+        EventStream stream = new EventStream();
+        stream.setStartEventId("startEventId");
+        stream.setEventTimestamp(CREATED_ON);
+        stream.setDaysSinceEvent(5);
+        stream.setStudyBurstId("studyBurstId");
+        stream.setStudyBurstNum(2);
+        stream.addEntry(6, makeDay("aaa"));
+        stream.addEntry(6, makeDay("bbb"));
+        stream.addEntry(8, makeDay("aaa"));
+        
+        JsonNode node = BridgeObjectMapper.get().valueToTree(stream);
+        assertEquals(node.size(), 8);
+        assertEquals(node.get("startEventId").textValue(), "startEventId");
+        assertEquals(node.get("eventTimestamp").textValue(), CREATED_ON.toString());
+        assertEquals(node.get("daysSinceEvent").intValue(), 5);
+        assertEquals(node.get("studyBurstId").textValue(), "studyBurstId");
+        assertEquals(node.get("studyBurstNum").intValue(), 2);
+        assertEquals(node.get("sessionGuids").size(), 2);
+        assertEquals(node.get("sessionGuids").get(0).textValue(), "aaa");
+        assertEquals(node.get("sessionGuids").get(1).textValue(), "bbb");
+        assertEquals(node.get("byDayEntries").get("6").get(0).get("type").textValue(), "EventStreamDay");
+        assertEquals(node.get("byDayEntries").get("6").get(1).get("type").textValue(), "EventStreamDay");
+        assertEquals(node.get("byDayEntries").get("8").get(0).get("type").textValue(), "EventStreamDay");
+        
+        EventStream deser = BridgeObjectMapper.get().readValue(node.toString(), EventStream.class);
+        assertEquals(deser.getStartEventId(), "startEventId");
+        assertEquals(deser.getDaysSinceEvent(), Integer.valueOf(5));
+        assertEquals(deser.getStudyBurstId(), "studyBurstId");
+        assertEquals(deser.getStudyBurstNum(), Integer.valueOf(2));
+        assertEquals(deser.getByDayEntries().size(), 2);
+        assertEquals(deser.getByDayEntries().get(6).size(), 2);
+        assertEquals(deser.getByDayEntries().get(8).size(), 1);
+    }
+    
+    @Test
+    public void retrieveEventStream() {
```

Comment:
I can't tell what this is testing. Is it just verifying that the setters take the expected input?

Reply:
It's testing nothing! Removed this.